src/client/styles/main.scss

```diff
@@ -54,6 +54,10 @@ button {
   @include box-sizing(border-box);
 }
 
+.hidden {
```

Comment:
Figured this was a bit of a shim because you mentioned your work in sunsetting .visuallyHidden.